uisdk/src/main/res/values-night/colors.xml

```diff
@@ -0,0 +1,140 @@
+    <color name="kh_uisdk_silver">@color/kh_uisdk_label</color>
+    <color name="kh_uisdk_light_grey">@color/kh_uisdk_background5</color>
+    <color name="kh_uisdk_lightish_grey">@color/kh_uisdk_background2</color>
+    <color name="kh_uisdk_lighter_grey">@color/kh_uisdk_background3</color>
+    <color name="kh_uisdk_off_white">@color/kh_uisdk_background1</color>
+    <color name="kh_uisdk_greyish_white">@color/kh_uisdk_background2</color>
+    <color name="kh_uisdk_opacity_off_white">@color/kh_uisdk_background2</color>
+    <color name="kh_uisdk_opacity_off_white_time_picker">@color/kh_uisdk_disabled</color>
+    <color name="kh_uisdk_transparent">#00000000</color>
+    <color name="kh_uisdk_transparent_black_map">#22000000</color>
+    <color name="kh_uisdk_transparent_black">#66000000</color>
+    <color name="kh_uisdk_transparent_black_notification">#AD000000</color>
+    <color name="kh_uisdk_background_light_grey">@color/kh_uisdk_background2</color>
+
+    <!-- Text -->
+    <color name="kh_uisdk_text_white">@color/kh_uisdk_background_primary</color>
+    <!--    <color name="kh_uisdk_text_primary">@color/kh_uisdk_newTextColor</color>-->
+    <color name="kh_uisdk_text_action_up">@color/kh_uisdk_dark_grey</color>
+    <color name="kh_uisdk_text_action_pressed">#450815</color>
+    <color name="kh_uisdk_text_alternative">#888888</color>
+    <color name="kh_uisdk_text_unavailable">@color/kh_uisdk_disabled</color>
+    <!--    <color name="kh_uisdk_text_green">@color/kh_uisdk_newSuccessColor</color>-->
+
+    <!-- Category Selector Divider -->
+    <color name="kh_uisdk_divider">#E2E2E2</color>
+
+    <!-- Tab Color Selector -->
+    <color name="kh_uisdk_tab_indicator">@color/kh_uisdk_colorAccent</color>
+
+    <!-- CANCEL BUTTON -->
+    <color name="kh_uisdk_transluscent_white">#CCFFFFFF</color>
+    <color name="kh_uisdk_transluscent_black">#CC000000</color>
+
+    <!-- RatingBar -->
+    <color name="kh_uisdk_duskYellow">#edd413</color>
+    <color name="kh_uisdk_lightGrey">#bab9b5</color>
+
+    <color name="kh_uisdk_ButtonDisabledColor">#66181447</color>
+    <color name="kh_uisdk_ButtonEnabledColor">@color/kh_uisdk_secondary</color>
+    <color name="kh_uisdk_TimePickerTitleBackgroundColor">@color/kh_uisdk_primary</color>
+    <color name="kh_uisdk_TimePickerTitleTextColor">@color/kh_uisdk_opacity_off_white_time_picker
+    </color>
+
+    <!-- Icon -->
+    <color name="kh_uisdk_icon_blue">#181A47</color>
+
+    <!-- Adyen -->
+    <!--    <color name="kh_uisdk_primaryColor">@color/kh_uisdk_primary</color>-->
+    <color name="kh_uisdk_loyalty_background">@color/kh_uisdk_background_primary</color>
+
+    <!-- New Colors -->
+    <color name="kh_uisdk_primary">#18183F</color>
+    <color name="kh_uisdk_secondary">#FF0082</color>
+    <color name="kh_uisdk_accent">#2485C2</color>
+<!--    <color name="kh_uisdk_accent_opacity50">#7F2485C2</color>-->
+    <color name="kh_uisdk_accent_opacity50">#E1F4FC</color>
```

Comment:
Can we add opacity to the color when we use it as opposed to declaring a new one?

Reply:
We can process the drawable's layer list but it adds extra complexity without reason. This is used in the filters wdiget and in the sort widget